livestyled/schemas/fulfilment_point.py

```diff
@@ -61,3 +62,4 @@ class Meta:
     categories = RelatedResourceField(schema=FulfilmentPointCategorySchema, many=True, missing=[])
     venue = RelatedResourceLinkField(schema=VenueSchema, required=False, missing=None)
     external_id = fields.String(missing=None, data_key='externalId')
+    audiences = RelatedResourceField(schema=AudienceSchema, many=True, missing=[])
```

Comment:
Have you tested this? I thought this should be `RelatedResourceLinkField`  like the others. please write a test to make sure this works.